NetKAN/World-Airforce-Insignia.netkan

```diff
@@ -0,0 +1,11 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "World-Airforce-Insignia",
+    "$kref"        : "#/ckan/github/chaseAEd/World-Airforce-Insignia",
+     "name"         : "World Airforce Insignia",
+    "abstract"     : "A simple mod that adds a range of current, or historic airforce roundels to add to your planes.",
+    "license"      : "MIT",
+    "ksp_version"  : "1.2.99",
```

Comment:
You'll need to promote from pre-release to release. CKAN doesn't index pre-releases. Also, if you attach a zip file with a stable folder structure (no version number in the folder name), it'll work much easier.